logging/loggingtest/logger.go

```diff
@@ -14,6 +14,11 @@ type logSubscription struct {
 	notify chan<- struct{}
 }
 
+type countRequest struct {
+	exp      string
```

Comment:
what is "exp"?
I think you can find a better name. Based on the use it could be "substr", but the question is on what data of the Request is that used?